### PR TITLE
[Datadog] Release CLI for GA

### PR DESCRIPTION
**Pull request by limingu**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [x] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Datadog

File: src/datadog/azext_datadog/generated/_params.py [hunk 1]
```diff
@@ -65,6 +65,9 @@ def load_arguments(self, _):
         c.argument('monitor_name', options_list=['--name', '-n', '--monitor-name'], type=str, help='Monitor resource '
                    'name', id_part='name')
         c.argument('tags', tags_type)
+        c.argument('monitoring_status', type=str, help='Flag specifying if the resource monitoring is enabled or '
```
Comment (jsntcy) [inline src/datadog/azext_datadog/generated/_params.py:68]: Do we need update tests as we added new arguments for command?

**Review by jsntcy — APPROVED** [review]

**Pull request merged**